binance_trade_bot/logger.py

```diff
@@ -4,53 +4,41 @@
 from .config import Config
 
 class Logger:
-
-    Logger = None
-    NotificationHandler = None
+    logger = None
 
     def __init__(self, config, logging_service="crypto_trading", enable_notifications=True):
-        # Logger setup
-        self.Logger = logging.getLogger(f"{logging_service}_logger")
-        self.Logger.setLevel(logging.DEBUG)
-        formatter = logging.Formatter("%(asctime)s - %(name)s - %(levelname)s - %(message)s")
-        # default is "logs/crypto_trading.log"
-        fh = logging.FileHandler(f"logs/{logging_service}.log")
-        fh.setLevel(logging.DEBUG)
-        fh.setFormatter(formatter)
-        self.Logger.addHandler(fh)
-
-        # logging to console
-        ch = logging.StreamHandler()
-        ch.setLevel(logging.INFO)
-        ch.setFormatter(formatter)
-        self.Logger.addHandler(ch)
-
-        # notification handler
-        self.NotificationHandler = NotificationHandler(enable_notifications)
-        self.notification_name = config.NOTIFICATION_NAME
-
-    def log(self, message, level="info", notification=True):
-
-        if level == "info":
-            self.Logger.info(message)
-        elif level == "warning":
-            self.Logger.warning(message)
-        elif level == "error":
-            self.Logger.error(message)
-        elif level == "debug":
-            self.Logger.debug(message)
-
-        if notification and self.NotificationHandler.enabled:
-            self.NotificationHandler.send_notification(f"```\n<{self.notification_name}>: {message}\n```")
-
-    def info(self, message, notification=True):
-        self.log(message, "info", notification)
-
-    def warning(self, message, notification=True):
-        self.log(message, "warning", notification)
-
-    def error(self, message, notification=True):
-        self.log(message, "error", notification)
-
-    def debug(self, message, notification=False):
-        self.log(message, "debug", notification)
+        self.logger = logging.getLogger(f"{logging_service}_logger")
+        self.logger.setLevel(logging.DEBUG)
+
+        stdout_formatter = logging.Formatter(
+            "%(asctime)s - %(name)s - %(levelname)s - %(message)s")
+        stdout_handler = logging.StreamHandler()
+        stdout_handler.setLevel(logging.DEBUG)
+        stdout_handler.setFormatter(stdout_formatter)
+        self.logger.addHandler(stdout_handler)
+
+        if enable_notifications:
+            try:
+                notification_formatter = logging.Formatter(
+                    f"```\n<{config.NOTIFICATION_NAME}>: %(asctime)s - %(name)s - %(levelname)s - %(message)s\n```")
```

Comment:
do we want all this or just the message for notifications?
e.g.
```suggestion
                    f"```\n<{config.NOTIFICATION_NAME}>: %(message)s\n```")
```

Reply:
Could drop the time since it comes up on Telegram anyway. Would like to keep the others though since eventually I want to have different loggers per module (or class if it makes sense) so you can tell where the log/error/whatever came from.

Reply:
```suggestion
                    f"```\n<{config.NOTIFICATION_NAME}>: %(name)s - %(levelname)s - %(message)s\n```")
```